src/components/ui-shell/ui-shell.config.js

```diff
@@ -37,6 +37,26 @@ const header = {
   ],
 };
 
+const headernav = {
```

Comment:
Would it help to add this onto `header` as another type? Maybe:

```js
const header = {
  company: 'IBM',
  platform: '[Platform]',
  links: [
    {
      href: '/component/ui-shell--default',
      title: 'L1 link 1',
    },
    {
      href: '/component/ui-shell--default',
      title: 'L1 link 1',
    },
    {
      type: 'menu',
      links: [
        // ...
      ],
    }
  ]
};
```

Then the template could support either type, similar to what we'll need to do in the React component.